### DISCUSSION TITLE
First Key on Keyboard reloads Organ

### DISCUSSION THREAD

**Discussion by larspalo** [GrandOrgue/grandorgue#527] [Help]
Originally posted 2018-08-30 15:39:12.480000 by jreichels in GrandOrgue discussion forum on SourceForge<br><br>Dear Gents,

When I connect my Viscount organ keyboard via Midi to GO (Win 10 Pro), then the first note on Key 36 will reload the Organ-File. The other keys (37 to 96) are shifted one note. The Midi-Log said, Key Note 36 was pressed.

Have somebody an idea, what the problem is? Thanks for helping.

Best regards

Jreichels

**Comment by larspalo** [comment]
Originally posted 2018-08-31 15:02:26.974000 by jreichels in GrandOrgue discussion forum on SourceForge<br><br>Dear Gents,
I solved the problem. I delete the cache an the configuration files and start over again. Now, the organ works fine.

Jreichels

**Comment by larspalo** [comment]
Originally posted 2018-09-01 17:25:45.161000 by [Martin Koegler](https://github.com/e9925248) in GrandOrgue discussion forum on SourceForge<br><br>You have probable activated the transpose function and configured the first keyboard key as MIDI event for loading that organ.

**Reply by JKBert** [reply]
Hello,
I have exactly the same trouble (GO 3.13.0 built from sources on a Linux Devuan box) but I have configured only one MIDI channel for keyboard. I haven't found option that reloads organ when first key is pressed.
I have tried also to delete cache and configuration files without result.
Help will be welcome.

**Comment by larspalo** [comment]
Originally posted 2018-09-24 19:00:48.771000 by briansweetnam in GrandOrgue discussion forum on SourceForge<br><br>I did not know that one could assign loading of organs to midi events.  That is great news indeed.
